app/models/event_type.rb

```diff
@@ -22,4 +24,8 @@ class EventType < ActiveRecord::Base
   def length_step
     errors.add(:length, "must be multiple of #{LENGTH_STEP}") if length % LENGTH_STEP != 0
   end
+
+  def capitalize_color
+    self.color = color.upcase if color.present?
```

Comment:
Agreed! @Ana06 could you please open an issue after we merged this PR. Could be a junior job....